### PR TITLE
Fixed #4616

### LINKED ISSUE #4616: Decrease dummy item to 499
* **rAthena Hash**:  47131b4
* **Client Date**:  -
* **Server Mode**: Pre/Re

* **Description of Issue**: 
  * Result: can`t load item with id lower then 501
  * Expected Result: item_db should be read items from item id: 500
change dummy_item->nameid = 499;
  * How to Reproduce: 
  * Official Information: kRo add new event item with id 500 [초코 강정]
https://www.divine-pride.net/database/item/500

* **Modifications that may affect results**:

### PR DESCRIPTION
**Pull request by vakhet**
* **Addressed Issue(s)**: #4616 

* **Server Mode**: Renewal

* **Description of Pull Request**: Changing dummy_item->nameid and adding new quest item

### CODE REVIEW

File: src/map/itemdb.cpp [hunk 1]
```diff
@@ -439,7 +439,7 @@ static void itemdb_create_dummy(void) {
 	CREATE(dummy_item, struct item_data, 1);
 
 	memset(dummy_item, 0, sizeof(struct item_data));
-	dummy_item->nameid = 500;
+	dummy_item->nameid = 499;
```
Comment (Lemongrass3110) [inline src/map/itemdb.cpp]: Can you please move the declaration to itemdb.hpp?
And call it `ITEMID_DUMMY`.
Thank you!

File: db/re/item_db.txt [hunk 2]
```diff
@@ -5,6 +5,7 @@
 //
 // Healing Items
 //=============================================================
+500,Choko_Gangjeong,Choko Gangjeong,0,0,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ percentheal 10,10; },{},{}
```
Comment (Lemongrass3110) [inline db/re/item_db.txt]: Weight should be 10.

File: src/map/itemdb.cpp (hunk 1 above)
Comment (vakhet) [reply src/map/itemdb.cpp]: Done. Nice tip!

File: src/map/itemdb.hpp [hunk 3]
```diff
@@ -7,6 +7,8 @@
 #include "../common/db.hpp"
 #include "../common/mmo.hpp" // ITEM_NAME_LENGTH
 
+///Minimum available Item ID - 1
+#define ITEMID_DUMMY 499
```
Comment (Lemongrass3110) [inline src/map/itemdb.hpp]: Check a few lines below. Should be part of the enum! :-)

Comment (vakhet) [reply src/map/itemdb.hpp]: Thought about that, but... "for hardcoded purpose"? Seems a bit vague .)
Do you mean to add new ITEMID_CHOKO_GANGJEONG, or ITEMID_DUMMY ? Or both?

Comment (vakhet) [reply src/map/itemdb.hpp]: O-ok. I've figure it out .)

**Pull request merged**